index.d.ts

```diff
@@ -34,18 +34,44 @@ declare namespace pug {
   export function compileFileClient(path: string, options?: Options): string;
 
   /**
+   * Compile a Pug template and render it without locals to html string.
+   */
+  export function render(template: string): string;
+  /**
    * Compile a Pug template and render it with locals to html string.
+   * @param {(Options & LocalsObject)} options Pug Options and rendering locals
```

Comment:
Typings has to be usefull, not look nice on github repository :stuck_out_tongue_winking_eye:  It's better to duplicate than have no info about params or function you are using.